src/main/java/com/direwolf20/buildinggadgets/common/tools/InventoryManipulation.java

```diff
@@ -24,14 +25,13 @@
 import java.util.stream.Stream;
 
 public class InventoryManipulation {
-    private static final Set<IProperty> safeProperties = Stream.of(BlockSlab.HALF, BlockStairs.HALF, BlockLog.LOG_AXIS,
-            BlockDirectional.FACING, BlockStairs.FACING, BlockTrapDoor.HALF, BlockTorch.FACING, BlockStairs.SHAPE, BlockLever.FACING, BlockLever.POWERED, BlockRedstoneRepeater.DELAY,
-            BlockStoneSlab.VARIANT, BlockWoodSlab.VARIANT, BlockDoubleWoodSlab.VARIANT, BlockDoubleStoneSlab.VARIANT).collect(Collectors.toSet());
+    private static IProperty AXIS = PropertyEnum.create("axis", EnumFacing.Axis.class);
+    private static final Set<IProperty> safeProperties = Stream.of(BlockSlab.HALF, BlockStairs.HALF, BlockLog.LOG_AXIS, AXIS,
```

Comment:
Is the BlockLog.LOG_AXIS still needed? (it isn't a standard axis Property...?)

Reply:
Yes, it's still needed because `BlockLog.LOG_AXIS` is created using `BlockLog.EnumAxis.class`, while `BlockLogsTC.AXIS` is created using `EnumFacing.Axis.class`.